server/index.js

```diff
@@ -423,12 +423,15 @@ app.prepare().then(() => {
   server.get('/blog/:name', (req, res) => app.render(req, res, '/post', { name: req.params.name }));
 
   server.get('/api/astronauts', async (req, res) => {
-    const team = await getTeam();
+
+    let team = await getTeam();
```

Comment:
@BohdanLeshchyshyn Why did you change it? If you don't change variables we must use `const`